### PR TITLE
Us133 borrower confirms deed

### PR DESCRIPTION
**Pull request by Lightman32**
Not ready to merge yet, just getting some code review early.

### CODE REVIEW

File: application/templates/viewdeed.html [hunk 1]
```diff
@@ -67,10 +67,16 @@ <h3 class="heading-medium">Effective date and time</h3>
 
         <h3 class="heading-medium">Borrower's digital signature(s)</h3>
         {% for borrower in deed_data.deed.borrowers %}
-          <p class="awaiting-information" id="signature{{ loop.index }}">[Awaiting digital signature
-            of {{ borrower.forename }}
-            {{ borrower.middle_name }}{% if borrower.middle_name %} {% endif %}{{ borrower.surname }}]
-          </p>
+            {% if borrower.signature %}
```
Comment (tsangsimon) [inline application/templates/viewdeed.html:70]: Can we sort out the indentation for the added code please.  bring the moustache condition block to be inline with the end of the p tag. use 2 spaces as well :)

Comment (danielmarker) [inline application/templates/viewdeed.html:70]: There is a way to set this in Pycharm which also reformats your code for
you - from experience :)

On 22 February 2016 at 15:32, Simon Tsang notifications@github.com wrote:

> In application/templates/viewdeed.html
> https://github.com/LandRegistry/dm-borrower-frontend/pull/43#discussion_r53640681
> :
> 
> > @@ -67,10 +67,16 @@ <h3 class="heading-medium">Effective date and time</h3>
> > 
> > ```
> >      <h3 class="heading-medium">Borrower's digital signature(s)</h3>
> >      {% for borrower in deed_data.deed.borrowers %}
> > ```
> > -          <p class="awaiting-information" id="signature{{ loop.index }}">[Awaiting digital signature
> > -            of {{ borrower.forename }}
> > -            {{ borrower.middle_name }}{% if borrower.middle_name %} {% endif %}{{ borrower.surname }}]
> > -          </p>
> > -            {% if borrower.signature %}
> 
> Can we sort out the indentation for the added code please. use 2 spaces as
> well :)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/LandRegistry/dm-borrower-frontend/pull/43/files#r53640681
> .

## 

_Dan Marker_
_Digital Mortgage Team - Land Registry_
_Information Systems_

**Comment by Kirosoft** [comment]
:+1:

**Comment by danielmarker** [comment]
:+1:

File: application/templates/viewdeed.html (hunk 1 above)
Comment (Lightman32) [inline application/templates/viewdeed.html:70]: Thanks, ^ issues fixed :)

**Comment by tsangsimon** [comment]
:+1:

**Pull request merged**